admin/views/dashboard-widget.php

```diff
@@ -13,15 +13,15 @@
  */
 
 ?>
-<p><?php _e( 'Below are your published posts&#8217; SEO scores. Now is as good a time as any to start improving some of your posts!', 'wordpress-seo' ); ?></p>
+<p><?php printf( __( 'Hey %s,', 'wordpress-seo' ), $user->user_firstname ); ?></p>
+<p><?php _e( 'Below are some posts that could use some work. Improve these posts if you want more traffic from the search engines:', 'wordpress-seo' ); ?></p>
 <table>
 	<?php foreach ( $statistics as $statistic ) :
-		if ( current_user_can( 'edit_others_posts' ) === false ) {
-			$url = esc_url( admin_url( 'edit.php?post_status=publish&post_type=post&seo_filter=' . $statistic['seo_rank'] . '&author=' . get_current_user_id() ) );
-		}
-		else {
-			$url = esc_url( admin_url( 'edit.php?post_status=publish&post_type=post&seo_filter=' . $statistic['seo_rank'] ) );
+		if ( $statistic['seo_rank'] === 'good' ) {
+			$good_posts = $statistic['count'];
```

Comment:
The value `$good_posts` is only been set here. What if there aren't any good posts? I would set a default with the value `0`